Tella/Scenes/Settings/Views/Servers/ServerSelectionView.swift

```diff
@@ -8,20 +8,17 @@
 
 import SwiftUI
 import Combine
+
 struct ServerSelectionView: View {
     @EnvironmentObject var serversViewModel : ServersViewModel
-    @ObservedObject var gDriveVM: GDriveAuthViewModel
     @ObservedObject var gDriveServerVM: GDriveServerViewModel
+    @ObservedObject var dropboxServerVM: DropboxServerViewModel
     
     @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
     
-    let gDriveRepository: GDriveRepositoryProtocol
-
-    
-    init(appModel:MainAppModel, server: TellaServer? = nil, gDriveRepository: GDriveRepositoryProtocol) {
-        self.gDriveRepository = gDriveRepository
-        _gDriveVM = ObservedObject(wrappedValue: GDriveAuthViewModel(repository: gDriveRepository))
+    init(appModel:MainAppModel, server: TellaServer? = nil, gDriveRepository: GDriveRepositoryProtocol, dropboxRepository: DropboxRepositoryProtocol) {
```

Comment:
What do you think of 

    init(appModel:MainAppModel, serversViewModel : ServersViewModel, gDriveRepository: GDriveRepositoryProtocol, dropboxRepository: DropboxRepositoryProtocol) {
        self.serversViewModel = serversViewModel
        _gDriveServerVM = ObservedObject(wrappedValue:GDriveServerViewModel(repository: gDriveRepository, mainAppModel: appModel))
        _dropboxServerVM = ObservedObject(wrappedValue: DropboxServerViewModel(dropboxRepository: dropboxRepository, mainAppModel: appModel))
    }


and update 

    @ObservedObject var serversViewModel : ServersViewModel